controllers/auth.js

```diff
@@ -1,10 +1,70 @@
+<<<<<<< Updated upstream
 exports.getLogin = (req, res, next) => {
+=======
```

Comment:
I don't think we need this here, but that depends on what we end up doing in the routes.  If we do need it, I think it should be moved below the imports for the user model and bcrypt